DataGenerator.py

```diff
@@ -0,0 +1,221 @@
+import pandas as pd
+import numpy as np
+from tqdm import tqdm
+import h5py
+import pickle
+from enum import Enum
+
+
+class KindOfData(Enum):
+    AUDIOSET = 1
+    LIVE = 2
+    ALL = 3
+
+
+class DataGenerator:
+    data_audioset = None
+    data_live = None
+    mask_audioset = None
+    mask_live = None
+
+    @classmethod
+    def get_generate(cls, type: KindOfData, interval: list):
```

Comment:
Please, rename the function to make it more clear. Something like `get_generated_sample` or `get_generated_sample_of_specific_type` etc
Moreover, please, add some kind of an error or assert for the case if not prepared data is requested. Maybe you can add a possibility to call data generation functions from here

Reply:
Done